wagtail/contrib/search_promotions/models.py

```diff
@@ -9,15 +9,31 @@
 
 
 class Query(models.Model):
+    """
+    Query model for search promotions
+
+    Attributes
+
+       - query_string (forms.CharField): Query text to match.
+    """
+
     query_string = models.CharField(max_length=MAX_QUERY_STRING_LENGTH, unique=True)
 
     def save(self, *args, **kwargs):
-        # Normalise query string
+        """
+        Save a normalised query string.
+        """
```

Comment:
```suggestion
        # Normalise query string
```

Please leave this line unchanged, it does not impact this feature.